internal/services/servicebus/servicebus_subscription_resource.go

```diff
@@ -129,6 +130,39 @@ func resourceServicebusSubscriptionSchema() map[string]*pluginsdk.Schema {
 				string(servicebus.EntityStatusReceiveDisabled),
 			}, false),
 		},
+
+		"client_scoped_subscription_enabled": {
+			Type:     pluginsdk.TypeBool,
+			Optional: true,
+			Default:  false,
+		},
+
+		"client_scoped_subscription": {
+			Type:     pluginsdk.TypeList,
+			Optional: true,
+			MaxItems: 1,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"client_id": {
+						Type:     pluginsdk.TypeString,
+						Optional: true,
+						ForceNew: true,
+					},
```

Comment:
should enabled be in the block?
```suggestion

		"client_scoped_subscription": {
			Type:     pluginsdk.TypeList,
			Optional: true,
			MaxItems: 1,
			Elem: &pluginsdk.Resource{
				Schema: map[string]*pluginsdk.Schema{
				
						"enabled": {
			Type:     pluginsdk.TypeBool,
			Optional: true,
			Default:  false,
		},
		
					"client_id": {
						Type:     pluginsdk.TypeString,
						Optional: true,
						ForceNew: true,
					},
```

Reply:
@katbyte Thanks for the comment, I left it outside of the block as I was considering that the two properties are better to exist parallel? Users only need to fill the client scoped properties if they choose to enable the feature.